mino/minows/rpc.go

```diff
@@ -0,0 +1,422 @@
+package minows
+
+import (
+	"context"
+	"encoding/gob"
```

Comment:
why are we using gob encoding everywhere instead of `serde` ? 🤔

Reply:
IIRC, the `serde` implementation used `json`, and as the first benchmarks were slow, it has been changed to a binary format. But in the end it turned out that the slowdown was due to another part of the implementation. So I think we can revert that, if you prefer to have `serde` everywhere.

Reply:
In fact he `gob` encodes the following structure:

```
type Packet struct {
	Source  []byte
	Payload []byte
}
```

And then uses `serde` to encode the payload.

Is it worth creating a `serde` abstraction for the `Packet`?

Reply:
To keep everything uniform, I'd say yes.

Reply:
OK, I looked at it, and I propose to keep the `Packet` as a gob-encoded structure, with the following reasoning:
- to me it corresponds to the grpc-encoding found here, which is encoded using protobuf (if I got that right): [overlayClient.Call](https://github.com/c4dt/dela/blob/main/mino/minogrpc/ptypes/overlay.pb.go#L427)
- as it needs to interface with the `Stream` interface, I'll have to add a packet-size manually, so that the receiver knows how many bytes to receive

So, I would be implementing part of the `gob` interface myself. Which is why I propose to accept this as-is.

@pierluca what do you think?